ansible/inventory/env/group_vars/all.yml

```diff
@@ -544,3 +544,8 @@ sunbird_desktop_keycloak_client_id: desktop
 sunbird_google_desktop_keycloak_client_id: google-auth-desktop
 sunbird_trampoline_android_keycloak_client_id: trampoline-android
 sunbird_trampoline_desktop_keycloak_client_id: trampoline-desktop
+
```

Comment:
Please move all these to your service env file. We dont need to define in `all.yml`